stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/typing/DefaultTypingUpdatesBuffer.kt

```diff
@@ -39,12 +40,7 @@ import kotlinx.coroutines.launch
  * @param onTypingStopped Signals that a stop typing event should be sent.
  * Usually used to make an API call using [io.getstream.chat.android.client.ChatClient.stopTyping]
  */
-@Deprecated(
-    message = "This class has been deprecated and will be removed." +
-        "Should you wish to implement your own buffer, you can create " +
-        "a custom implementation of `TypingUpdatesBuffer`.",
-    level = DeprecationLevel.ERROR,
-)
+@InternalStreamChatApi
 public class DefaultTypingUpdatesBuffer(
```

Comment:
Just wondering if it was left intentionally, cause as per the `Deprecated` description it was supposed to be deleted.

Reply:
Yup, this was intentionally left here.
The point was simply to obscure it from the end user so we can have tighter control over it in the future.

We use it internally so there are no actual plans to remove it, we just need to make users aware of the fact that we're not putting it out for public use, hence the `@InternalStreamChatApi` annotation.